src/Arcus.EventGrid/Contracts/RawEvent.cs

```diff
@@ -1,51 +1,53 @@
 ﻿using System;
 using GuardNet;
-using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Arcus.EventGrid.Contracts
 {
-    public class RawEvent : Event<object>
+    /// <summary>
+    /// Representation of an event with raw JSON content.
+    /// </summary>
+    public class RawEvent : EventGridEvent<object>
     {
+        private RawEvent() { }
+
+        /// <summary>
+        ///     Constructor
+        /// </summary>
+        /// <param name="id">Id of the event</param>
+        /// <param name="eventType">Type of the event</param>
+        /// <param name="data">Body of the event</param>
+        /// <param name="subject">Subject of the event</param>
+        /// <param name="dataVersion">Data version of the event body</param>
+        /// <param name="eventTime">Time when the event occured</param>
+        public RawEvent(string id, string eventType, string data, string subject, string dataVersion, DateTimeOffset eventTime) 
+            : this(id, eventType, data, subject, dataVersion, eventTime.DateTime) { }
+
         /// <summary>
         ///     Constructor
         /// </summary>
         /// <param name="id">Id of the event</param>
-        /// <param name="type">Type of the event</param>
-        /// <param name="body">Body of the event</param>
+        /// <param name="eventType">Type of the event</param>
+        /// <param name="data">Body of the event</param>
         /// <param name="subject">Subject of the event</param>
         /// <param name="dataVersion">Data version of the event body</param>
         /// <param name="eventTime">Time when the event occured</param>
-        public RawEvent(string id, string type, string body, string subject, string dataVersion, DateTimeOffset eventTime)
+        public RawEvent(string id, string eventType, string data, string subject, string dataVersion, DateTime eventTime) : base(id, dataVersion, eventType)
```

Comment:
Hm why are we moving away from `DateTimeOffset`?

Reply:
The parent class also has an `EventTime` property which is a `DateTime`. I implemented the `IEvent` explicitly to not break that contract in the parent class.

Reply:
I hate `DateTime` as it sucks in terms of timezones (😁) but can see your point.

Maybe we can provide both?

Not required, feel free to resolve if you don't see the benefit

Reply:
I had this first added this way (both) but since it would conflict with a serialized version with "duplication" of data I think it would clutter only. 
A shame really.

Reply:
Frankly, then I would only use DateTimeOffset, but i'll leave it up to you to decide as I'm open for both.

Reply:
I'm resolving this, current approach is fine!